README.md

```diff
@@ -621,6 +621,11 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [preflight.com](https://preflight.com) - No-code automated web testing. Record tests on your browser that are resilient to UI changes and run them on Windows machines. Could you integrate with your CI/CD? The free plan includes 50 monthly test runs with video, HTML sessions, and more.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app, and unlimited projects, 5,000 snapshots/month.
   * [lost-pixel.com](https://lost-pixel.com) - holistic visual regression testing for your Storybook, Ladle, Histoire stories and Web Apps. Unlimited team members, totally free for open-source, 7,000 snapshots/month.
+  * [Requestly](https://requestly.com/)
+    * [Web Debugger](https://requestly.com/products/web-debugger/) — Open-source Chrome Extension to Intercept, Redirect and Mock HTTP Requests. Redirect URLs, Modify HTTP Headers, Mock APIs, Inject custom JS, Modify GraphQL Requests. Alternative to Charles Proxy & Fiddler. Free tier include creating upto 5 rules. Completely free for open source projects.
```

Comment:
Hard to say about these since its not clear whats SaaS and whats not.

Regardless, please fold to one line.

something like:

```
 * [Requestly](https://requestly.com/) Web developer suite featuring [Debugger](...), [Mock Server](....), [API Client](...) and [Session Recorder](....).  Small blurp here describing the suite goals.  Small statement about it being free with limits.
```

The list is huge, so we try to limit multi entries to only huge offerings.  I plan to edit out the remaining few small but multi line ones.

Reply:
> The list is huge, so we try to limit multi entries to only huge offerings. 

Makes sense. Folded ours to single line.

> Does this interact with the SaaS?

Yes, the Chrome Extension is controlled by a SaaS based Dashboard (in case you'd like to try - https://app.requestly.io/)